### DISCUSSION TITLE
[Q] What do these colors mean

### DISCUSSION THREAD

**Discussion by yzkno1** [exceljs/exceljs#2405] [Q&A]
## 💬 What do these colors mean
<img width="1520" alt="WeChatbde463e4f89723a4a44d99e1cb0c4218" src="https://user-images.githubusercontent.com/42439916/97130539-a82f7c80-177c-11eb-96a9-3b43086de7de.png">
<img width="1156" alt="WeChatccb9a45989ec495435876866ca335759" src="https://user-images.githubusercontent.com/42439916/97130597-d1500d00-177c-11eb-81d1-4f2414316d9d.png">

What do these colors mean

**Comment by mfahmirukman** [comment]
from the documentation, fgColor refers to the foreground color of the cell. bgColor refers to the background color of the cell

**Comment by Ellean** [comment]
What does `indexed: 64` means ?
In other word, how could I convert it to `RBG` ?

**Comment by trsiddiqui** [comment]
Did you find an answer?
